### ISSUE TITLE
xchtmlreport generation only shows failed cases from last simularor

### ISSUE THREAD

**Issue by jineshqa** [lyndsey-ferguson/fastlane-plugin-test_center#137]
### New Issue Checklist

- [X] Updated `fastlane-plugin-test_center` to the latest version
- [X] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [X] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [X] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description

1. I am running multiscan with 3 simulators in parallel
2. Of the 3 simulator runs only failures from last worker are logged in summaries
3. As a result when I try to create HTML output using xchtmlreport failures from last worker are diplayed but other failures are not displayed correctly
4. Our discussion [issue-107](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/issues/107#issuecomment-520640116)

##### Complete output when running fastlane, including the stack trace and command used
I am attaching a zip file which has console1.log with fastlane output, test summary, html report generated by this plugin and html report generated by xchtmlreport.
[test_output.zip](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/files/3561682/test_output.zip)

Also, if you want I could upload the project on github.

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.14.5                                                        |
| Ruby                        | 2.3.7                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.19.0                                             |
| Installation Source         | ~/.gems/bin/fastlane                                           |
| Host                        | Mac OS X 10.14.5 (18F132)                                      |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib |
| OpenSSL Version             | LibreSSL 2.6.5                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 10.2.1                                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.130.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                      | Version | Update-Status |
| --------------------------- | ------- | ------------- |
| fastlane-plugin-test_center | 3.8.3   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                         | Version      |
| --------------------------- | ------------ |
| did_you_mean                | 1.0.0        |
| slack-notifier              | 2.3.2        |
| rouge                       | 2.0.7        |
| xcpretty                    | 0.3.0        |
| terminal-notifier           | 2.0.0        |
| terminal-table              | 1.8.0        |
| multipart-post              | 2.0.0        |
| word_wrap                   | 1.0.0        |
| public_suffix               | 2.0.5        |
| babosa                      | 1.0.2        |
| colored                     | 1.2          |
| highline                    | 1.7.10       |
| commander-fastlane          | 4.4.6        |
| http-cookie                 | 1.0.3        |
| faraday-cookie_jar          | 0.0.6        |
| gh_inspector                | 1.1.3        |
| mini_magick                 | 4.9.5        |
| multi_xml                   | 0.6.0        |
| security                    | 0.1.3        |
| xcpretty-travis-formatter   | 1.0.0        |
| naturally                   | 2.2.0        |
| simctl                      | 1.6.5        |
| jwt                         | 2.1.0        |
| uber                        | 0.1.0        |
| declarative                 | 0.0.10       |
| declarative-option          | 0.1.0        |
| representable               | 3.0.4        |
| retriable                   | 3.1.2        |
| mime-types                  | 3.2.2        |
| httpclient                  | 2.8.3        |
| google-api-client           | 0.23.9       |
| tty-cursor                  | 0.7.0        |
| tty-spinner                 | 0.9.1        |
| tty-screen                  | 0.7.0        |
| faraday                     | 0.15.4       |
| json                        | 2.2.0        |
| io-console                  | 0.4.5        |
| excon                       | 0.66.0       |
| rubyzip                     | 1.2.3        |
| CFPropertyList              | 3.0.1        |
| libxml-ruby                 | 2.9.0        |
| plist                       | 3.5.0        |
| atomos                      | 0.1.3        |
| colored2                    | 3.1.2        |
| nanaimo                     | 0.2.6        |
| xcodeproj                   | 1.12.0       |
| claide                      | 1.0.3        |
| google-cloud-env            | 1.2.1        |
| google-cloud-core           | 1.3.1        |
| digest-crc                  | 0.4.1        |
| google-cloud-storage        | 1.16.0       |
| memoist                     | 0.16.0       |
| multi_json                  | 1.13.1       |
| googleauth                  | 0.6.7        |
| unf                         | 0.1.4        |
| domain_name                 | 0.5.20190701 |
| faraday_middleware          | 0.13.1       |
| signet                      | 0.11.0       |
| fastimage                   | 2.1.5        |
| bundler                     | 2.0.2        |
| unicode-display_width       | 1.6.0        |
| addressable                 | 2.6.0        |
| os                          | 1.0.1        |
| psych                       | 2.1.0.1      |
| mime-types-data             | 3.2019.0331  |
| xctest_list                 | 1.1.8        |
| colorize                    | 0.8.1        |
| fastlane-plugin-test_center | 3.8.3        |
</details>

*generated on:* **2019-08-30**
</details>

**Comment by jineshqa** [comment]
@lyndsey-ferguson apologies for the delay. This ticket has all the detail that you might need also the total # of failing cases is 5 (mentioned below) vs the html report which shows 2 failures.

testExample6
testExample25
testExample26
testExample33
testExample38

**Comment by lyndsey-ferguson** [comment]
This is a great example of how to log a bug. Thank you so much. I will try and get to it after I solve an already-in-progress investigation.

**Comment by jineshqa** [comment]
I'm humbled with your kind words. Thank you for everything you do.

**Comment by lyndsey-ferguson** [comment]
Xcode 11 adds support for merging test results. I won't spend time on adding support for collating test result bundles when people can just upgrade to Xcode 11.

**Comment by jineshqa** [comment]
Could you share how to collate results with xcode 11?

**Comment by lyndsey-ferguson** [comment]
`xcrun xcresulttool merge bundle1 bundle2 --output-path merged-bundle`

**Comment by lyndsey-ferguson** [comment]
I'm still waiting for your findings for my fix on https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/issues/145

**Comment by jineshqa** [comment]
@lyndsey-ferguson Is this good?

[console_verbose.log](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/files/3703785/console_verbose.log)
